pandas/tests/arrays/categorical/test_api.py

```diff
@@ -337,11 +337,28 @@ def test_remove_categories(self):
         # inplace == True
         res = cat.remove_categories("c", inplace=True)
         tm.assert_categorical_equal(cat, new)
+
         assert res is None
 
-        # removal is not in categories
-        with pytest.raises(ValueError):
-            cat.remove_categories(["c"])
+    @pytest.mark.parametrize(
+        "message, old, category", [
```

Comment:
no need to parametrize `message` and `old`. and I would rename `category` to `removals` to be consistent with the argument name, see https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.CategoricalIndex.remove_categories.html

Reply:
`old` is needed to create the categorical list `cat` and `msg` to check the error message.

Reply:
the values for `old` and `msg` are the same for all the tests. so no need to parametrize.